api/domain/ordersDomain.js

```diff
@@ -0,0 +1,50 @@
+const orderRepo = require('@repos/orderRepository');
+
+const { 
+    IdNotFound, 
+    SequelizeError } = require('@helpers/errors');
+
+exports.create = async (orderData) => {
+    try {
+        return await orderRepo.createOrder(orderData);
+    } catch (error) {
+        throw new Error(error);
+    }
+};
+
+exports.update = async (orderId, updatedOrderData) => {
+    try {
+        await orderRepo.getById(orderId);
```

Comment:
what is the point of this line?

Reply:
and what do u mean?

Reply:
why are you calling await orderRepo.getById(orderId);